client/verta/verta/_internal_utils/model_dependencies.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+
+import ast
+import inspect
+import re
+from importlib.metadata import packages_distributions
```

Comment:
Don't forget we'll need to add [`importlib-metadata`](https://pypi.org/project/importlib-metadata/) as [a client dependency](https://github.com/VertaAI/modeldb/blob/main/client/verta/requirements.txt). Looks like the version we need for `packages_distributions` is [`>=3.7.0`](https://github.com/python/importlib_metadata/blob/main/CHANGES.rst#v370)

Reply:
Agreed.  I'm actually dropping this out of this PR, since it is more related to the next batch of work anyway.  I put a comment on that Jira issue so we won't forget.